test/ClangImporter/pch-bridging-header.swift

```diff
@@ -1,7 +1,19 @@
-// RUN: rm -rf %t && mkdir -p %t
+// REQUIRES: objc_interop
+// RUN: rm -rf %t && mkdir -p %t/tmp
+
+// First test the explicit frontend-based bridging PCH generation and use works
```

Comment:
Please add tests under test/Driver/ using `-driver-print-actions` and `-driver-print-jobs`, so that we can see the individual steps being generated properly.